Software/bsw/static/Mcal/CAN/src/Can.c

```diff
@@ -406,7 +405,7 @@ void Can_Init( const Can_ConfigType* Config)
                    /* Do Nothing */
                 }
             #endif
-            MessageObjAssignedToHTH[HTHCount].HTHId = Global_Config->CanHardwareObjectRef[HOHCount].CanObjectId ;
+            MessageObjAssignedToHTH[HTHCount].HTHIndex = HOHCount;
```

Comment:
Could you please tell me what happened here?

Reply:
I just found that I need to know the index of the HOH configuration saved in the hardwareObject array we made in the PBcfg.c instead of the HOH ID .. it's more helpful in coding